bin/common/usr_local_bin_to_path.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Add /usr/local/bin to non-login path, since tools are installed there
+if ! grep ENV_SUPATH /etc/login.defs | grep -q /usr/local/bin; then
+  sed -i '/ENV_SUPATH/s/$/:\/usr\/local\/bin/' /etc/login.defs
```

Comment:
Based my understanding, the privileged provisioning commands are run in shells which are both non-login, and non-interactive. `/etc/profile` still gets sourced on non-interactive login shells, but when you run something with sudo, profile and rc scripts are not sourced because it's not a login shell. Otherwise `/etc/profile` would probably be a better place to modify the path.